R/data.table.R

```diff
@@ -2880,7 +2882,7 @@ isReallyReal = function(x) {
       ## redirect to normal DT[x == TRUE]
       stub = call("==", as.symbol(col), TRUE)
     }
-    if (length(stub[[1L]]) != 1) return(NULL) ## Whatever it is, definitely not one of the valid operators
+    if (length(stub[[1L]]) != 1) return(NULL) # nocov Whatever it is, definitely not one of the valid operators
```

Comment:
I can't see a way to reach this branch and the comment suggests it's there as an internal default, could you confirm @MarkusBonsch?

Reply:
You are completely right.